src/main/io/gps.c

```diff
@@ -252,11 +254,50 @@ void gpsInit(serialConfig_t *initialSerialConfig, gpsConfig_t *initialGpsConfig)
 
 void gpsInitNmea(void)
 {
+#ifdef COLIBRI_RACE
+    uint32_t now;
+#endif
+
     switch(gpsData.state) {
         case GPS_INITIALIZING:
+#ifdef COLIBRI_RACE
```

Comment:
this code just set the GPS go to 115200bps, maybe we can make it configurable in the cleanflight configurator later on.